src/core/shape_inference/include/irdft_shape_inference.hpp

```diff
@@ -0,0 +1,140 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#pragma once
+#include <openvino/op/irdft.hpp>
+
+#include "openvino/core/axis_vector.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace op {
+namespace util {
+template <class T>
+void irdft_shape_infer(const ov::op::v9::IRDFT* op,
```

Comment:
This method looks very similar to rdft_shape_infer() except for minor changes like different axes normalizing. Can we move common code to single place?

Reply:
Done.